backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Uni;
+import org.jboss.resteasy.reactive.RestQuery;
+
+import javax.inject.Inject;
+import javax.persistence.NoResultException;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import static com.redhat.cloud.notifications.Constants.INTERNAL;
+
+@Path(INTERNAL + "/validation")
```

Comment:
I'm wondering if we should move this to the Internal file or create a new `internal` submodule (i.e. `routers/internal`)  and put all our internal API's in there

Reply:
I was wondering the same. Whereas I don't like the grouping by internal. This probably leads to putting things that don't belong together from a domain perspective in the same class.
The same as using Util classes. Usually these classes belong pretty much everything and the methods inside utils classes have nothing in common.